src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2090,10 +2090,9 @@ def config_diagnostics(cmd, resource_group_name, name, level=None,
     if application_logging:
         fs_log = None
         blob_log = None
-        level = application_logging != 'off'
+        level = level if application_logging != 'off' else False
```

Comment:
A code style issue. The name `level` indicates that it is an enum or string type. Possible values can be "INFO", "DEBUG", etc. In this line, it is a boolean type. Does it represent turn logging on or off?

Reply:
> Just test with optional -level before merging

I have tested with optional `-level` it is not causing any exception. But it is not making any change to the log configuration. Current command will set it as `Verbose` by default. I did the same changes and tested in all the possible ways. Now it is working as expected.

Reply:
> A code style issue. The name `level` indicates that it is an enum or string type. Possible values can be "INFO", "DEBUG", etc. In this line, it looks like a boolean type. Does it represent turn logging on or off?

yes if user is not given the `-level` parameter level becomes `True` and it sets `Verbose`  level by default and when user want to `off` the logging then level becomes `false` and makes all the logging configurations `off`.